Define what are Endpoints (#11692)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint.md

Microsoft Defender for Endpoint is an enterprise endpoint security platform designed to help enterprise networks prevent, detect, investigate, and respond to advanced threats.

> Endpoints are:  Laptops, Phones, Tablets, PCs, Access Points, Routers, Firewalls? //List out what Microsoft considers Endpoints//

Reviewer: ```suggestion
> [!NOTE]
> Endpoints are laptops, phones, tablets, PCs, access points, routers, and firewalls.
```
